posts/2023-07-08-boa-release-17.md

```diff
@@ -0,0 +1,379 @@
+---
+layout: post
+tags: post
+title: "Boa release v0.17"
+author: Boa Developers
+---
+
+## Summary
+
+Boa v0.17 is now available! This is one of the biggest Boa releases since the project started, and after around 7 months
+of development, we are very happy to present you the latest release of the Boa JavaScript engine. Boa makes it easy to
+embed a JS engine in your projects, and you can even use it from WebAssembly. See the [about](/about) page for more info.
+
+In this release, our conformance has grown from 74.53% to 78.74% in the official ECMAScript Test Suite (Test262). While
+this might look as a small increase, we now pass **6,079** more tests than in the previous version. In any case, the big
+changes in this release are not related to conformance, but to huge internal enhancements and new APIs that you will be
+able to use.
+
+You can check the full list of changes [here][changelog], and the full information on conformance [here][conformance].
+
+Moreover, this big release was partly possible thanks to a grant by [Lit Protocol](https://litprotocol.com/). Lit is
+distributed key management for encryption, signing, and compute. Thanks to this grant, we were able to remunerate 2 team
```

Comment:
I think we can omit explaining what lit protocol is, the hyperlink takes care of that for us.

Reply:
Done :)